kibble/cli/setup_command.py

```diff
@@ -0,0 +1,197 @@
+
+    # Check if index already exists
+    if es.indices.exists(dbname + "_api"):
+        # Skip this is DB exists and -k added
+        if skiponexist:
+            print("DB prefix exists, but --skiponexist used, skipping this step.")
+            return
+        print("Error: ElasticSearch DB prefix '%s' already exists!" % dbname)
+        sys.exit(-1)
+
+    types = [
+        "api",
+        # ci_*: CI service stats
+        "ci_build",
+        "ci_queue",
+        # code_* + evolution + file_history: git repo stats
+        "code_commit",
+        "code_commit_unique",
+        "code_modification",
+        "evolution",
+        "file_history",
+        # forum_*: forum stats (SO, Discourse, Askbot etc)
+        "forum_post",
+        "forum_topic",
+        # GitHub stats
+        "ghstats",
+        # im_*: Instant messaging stats
+        "im_stats",
+        "im_ops",
+        "im_msg",
+        "issue",
+        "logstats",
+        # email, mail*: Email statitics
+        "email",
+        "mailstats",
+        "mailtop",
+        # organisation, view, source, publish: UI Org DB
+        "organisation",
+        "view",
+        "publish",
+        "source",
+        # stats: Miscellaneous stats
+        "stats",
+        # social_*: Twitter, Mastodon, Facebook etc
+        "social_follow",
+        "social_followers",
+        "social_follower",
+        "social_person",
+        # uisession, useraccount, message: UI user DB
+        "uisession",
+        "useraccount",
+        "message",
+        # person: contributor DB
+        "person",
+    ]
+
+    for t in types:
+        iname = f"{dbname}_{t}"
+        print(f"Creating index {iname}")
+
+        settings = {"number_of_shards": shards, "number_of_replicas": replicas}
+        es.indices.create(
+            index=iname, body={"mappings": mappings["mappings"], "settings": settings}
+        )
+    print(f"Indices created!")
+    print()
+
+    salt = bcrypt.gensalt()
+    pwd = bcrypt.hashpw(admin_pass.encode("utf-8"), salt).decode("ascii")
+    print("Creating administrator account")
+    doc = {
+        "email": admin_name,  # Username (email)
+        "password": pwd,  # Hashed password
+        "displayName": "Administrator",  # Display Name
+        "organisations": [],  # Orgs user belongs to (default is none)
+        "ownerships": [],  # Orgs user owns (default is none)
+        "defaultOrganisation": None,  # Default org for user
+        "verified": True,  # Account verified via email?
+        "userlevel": "admin",  # User level (user/admin)
+    }
+    dbdoc = {
+        "apiversion": KIBBLE_VERSION,  # Log current API version
+        "dbversion": KIBBLE_DB_VERSION,  # Log the database revision we accept (might change!)
+    }
+    es.index(index=dbname + "_useraccount", doc_type="_doc", id=admin_name, body=doc)
+    es.index(index=dbname + "_api", doc_type="_doc", id="current", body=dbdoc)
+    print("Account created!")
+
+
+def run(
```

Comment:
```suggestion
def do_setup(
```
How about this name?